Prace_domowe/Praca_domowa4/Gruszkowska_Klaudia/PD 4.ipynb

```diff
@@ -0,0 +1,1760 @@
+{
```

Comment:
<p>de facto jest to RMSE a nie MSE</p><p> </p><p>RMSE warto odnieść do średniej wartości targetu aby wiedzieć czy błąd jest duży czy nie.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/211/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>